packages/ebayui-core/src/components/ebay-snackbar-dialog/component.ts

```diff
@@ -5,7 +5,8 @@ import type { Input as BaseInput } from "../components/ebay-dialog-base/componen
 interface SnackbarDialogInput extends Omit<BaseInput, `on${string}`> {
     layout?: "row" | "column";
     action?: BaseInput["action"] & {
-        accessKey?: string;
+        accesskey?: string;
+        accessKey?: string; // deprecated
```

Comment:
Can you use JSDoc here, so people see deprecations from their editor?
```suggestion
        /** @deprecated use `accesskey` instead */
        accessKey?: string;
```